src/ExtractToFStar.ml

```diff
@@ -1550,15 +1553,18 @@ let extract_global_definition (ctx : extraction_ctx) (fmt : F.formatter)
 let extract_global_decl (ctx : extraction_ctx) (fmt : F.formatter)
   (qualif : fun_decl_qualif) (def : fun_decl)
     : unit =
+  (* TODO Lookup LLBC decl *)
```

Comment:
My proposal is the other way around: you receive a `global_decl` as input, and lookup the `fun_decl` corresponding to the input